scripts/build-flutter-app.sh

```diff
@@ -5,15 +5,13 @@ set -e
 # Display each command as it's run.
 set -x
 
-DEPLOY_BRANCH=deploy-slac
-
 cd /usr/src/rubintv
 git clone --single-branch --branch $DEPLOY_BRANCH https://github.com/lsst-sitcom/rubin_chart
 git clone --single-branch --branch $DEPLOY_BRANCH https://github.com/lsst-ts/rubintv_visualization ./ddv
```

Comment:
If this is going to be set by environment var, would it be useful to have a more specified name, i.e. `DDV_DEPLOY_BRANCH`?

Reply:
good idea! that would be better. I'll make the change :+1:.